<launch>
    <node pkg="verbal_communication" type="SpeechObserver.py" name="SpeechObserver" output="screen">
        <param name="robot_name" value="terminator" />
    </node>
    <node pkg="verbal_communication" type="Encoder.py" name="Encoder" output="screen" />
    <node pkg="verbal_communication" type="Speaker.py" name="Speaker" output="screen" />
    <node pkg="verbal_communication" type="GreetingTask.py" name="Greeting" output="screen">
        <param name="robot_name" value="terminator" />
    </node>

</launch>